browser/ui/views/brave_default_browser_dialog_view.h

```diff
@@ -0,0 +1,42 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_BRAVE_DEFAULT_BROWSER_DIALOG_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_BRAVE_DEFAULT_BROWSER_DIALOG_VIEW_H_
+
+#include <memory>
+
+#include "ui/views/window/dialog_delegate.h"
+
+namespace views {
+class Label;
+}  // namespace views
+
+class BraveDefaultBrowserDialogView : public views::DialogDelegateView {
```

Comment:
since recently, I'm pretty paranoid re accessibility properties of views :) It looks good here, but maybe you also could double-check

Reply:
I think it's fine because this dialog doesn't introduce new view class.
and did test many times with virtual keyboard on. :)